### ISSUE TITLE
Can't edit time & duration in List view?

### ISSUE THREAD

**Issue by HolySamosa** [toggl/track-macos-feedback#45]
## Summary

- [ ] Crash 
- [X] Glitch 
- [ ] Sync 

The time and duration of my entries don't appear in the "Detail" pop-up of the list view, as they did with the old app.  So I can't edit the time information of completed entries-- which is absolutely essential.  I can only assume something is broken here?

## Steps to reproduce

1. Double click an item in the list view.
2. No time or duration is visible.  (See image)

![image](https://user-images.githubusercontent.com/1156546/222199220-aac1921b-298d-488a-b932-37e639f1ab28.png)

## Expected behavior

The start time and duration are presented and can be edited within the "Detail" view.

## Affected versions

**App version:**  8.9.0 (2066)
**OS version:** OS X
**Device model:**

**Comment by AndrewVebster** [comment]
Hey @HolySamosa 👋 

From your screenshot I can tell that you're trying to edit the group of three time entries. You can see this by the number on the left - it means that three time entries are grouped together because they are similar. This way changing the description/project/tag/billable here would apply those changes to all time entries in a group.

To edit a single entry, please click on the number button on the left. This will expand the group and you can edit a single entry.

![CleanShot 2023-03-01 at 18 34 59@2x](https://user-images.githubusercontent.com/3470113/222204028-352dd570-4591-4cdd-82df-0682b8fa9e1a.png)

If you don't want to have your entries grouped, you can disable this option via iOS or Web application (this setting will be added to macOS soon, so sorry for that).

It's the screen from the web application: go to Profile Settings 
![CleanShot 2023-03-01 at 18 33 32@2x](https://user-images.githubusercontent.com/3470113/222204249-21f68b4c-7916-42e5-8507-eeb366f68a4f.png)

**Comment by HolySamosa** [comment]
Thanks, @AndrewVebster.  Yep, I totally missed the grouping!

I think some minor UI changes could improve the UX and make the grouping more apparent.   A couple of thoughts:
- In the Details pop-up, a heading of "**Details (3 items)**" instead of "**Details (3)**"
- For a new user, there isn't an indication that a list item is expandable.  Once the interface is learned, the circled number provides this indication but this isn't a standard indicator for this kind of interaction.  It could more clearly indicate the item is expandable if a standard symbol / icon (such as arrows / chevrons) for this interaction were added to the left edge, along with the circled number.  For example:
![image](https://user-images.githubusercontent.com/1156546/222488206-df53cc20-4fb0-4687-8204-8a0fedbe4b55.png)

I also *really* hope the option to enable / disable grouping is added to the OS X app ASAP.  For me personally, this is vital to how I use Toggl.  However, I see that grouping would be incredibly helpful for certain tasks-- but for me it would be important to easily and quickly switch between the two.  So rather than hide this option in the Preferences section, allow this to be easily enabled / disabled directly from the list view.

I've been using Toggl almost as long as it's been publically available and I've recommended it to many people-- especially for students and professionals with ADHD.  To me, **Toggl's greatest strength vs other options has always been the simplicity of the tracker app.**  The UI been incredibly intuitive, user can get the mental model within a couple of minutes, making in incredibly easy to immediately start using it in a productive way, without requiring effort to learn.  I really that as new features are added, this simplicity (IMHO, your secret sauce) isn't lost.

I think this last point is so important, that I kind of feel the need to create a separate issue just to give it proper attention...

Thank you!
